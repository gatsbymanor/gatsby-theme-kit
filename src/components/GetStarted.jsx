import React from 'react';

export default function GetStarted(props) {
  return (
    <section id="cta" className="main special">
      <header className="major">
        <h2>Congue imperdiet</h2>
        <p>Donec imperdiet consequat consequat. Suspendisse feugiat congue<br />
          posuere. Nulla massa urna, fermentum eget quam aliquet.</p>
      </header>
      <footer className="major">
        <ul className="actions">
          <li><a href="generic.html" className="button special">Get Started</a></li>
          <li><a href="generic.html" className="button">Learn More</a></li>
        </ul>
      </footer>
    </section>
  )
}