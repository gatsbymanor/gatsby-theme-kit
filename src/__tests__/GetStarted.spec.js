import React from 'react';
import renderer from 'react-test-renderer';
import GetStarted from '../components/GetStarted';

describe('<GetStarted />', () => {
  const component = shallow(<GetStarted />);

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});