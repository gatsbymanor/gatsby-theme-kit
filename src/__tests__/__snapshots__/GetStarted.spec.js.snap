// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetStarted /> should render correctly 1`] = `
<section
  className="main special"
  id="cta"
>
  <header
    className="major"
  >
    <h2>
      Congue imperdiet
    </h2>
    <p>
      Donec imperdiet consequat consequat. Suspendisse feugiat congue
      <br />
      posuere. Nulla massa urna, fermentum eget quam aliquet.
    </p>
  </header>
  <footer
    className="major"
  >
    <ul
      className="actions"
    >
      <li>
        <GatsbyLink
          className="button special"
          to="/generic"
        >
          Get Started
        </GatsbyLink>
      </li>
      <li>
        <GatsbyLink
          className="button"
          to="/generic"
        >
          Learn More
        </GatsbyLink>
      </li>
    </ul>
  </footer>
</section>
`;
